### PR TITLE
Upgrade `@stylable/dom-test-kit`

### PR DESCRIPTION
**Pull request by NitayRabi**
- [x] Update @stylable versions 
- [x] Migrate to `Compt` mode

### CODE REVIEW

File: packages/wix-ui-core/package.json [hunk 1]
```diff
@@ -65,7 +65,8 @@
     "update-components": ".wuf/update-components.sh"
   },
   "dependencies": {
-    "@stylable/dom-test-kit": "^0.1.0",
+    "@stylable/dom-test-kit": "^2.1.5-alpha.1",
+    "@stylable/jest": "^2.1.8-alpha.1",
```
Comment (shlomitc) [inline packages/wix-ui-core/package.json]: why is this needed?

**Comment by NitayRabi** [comment]
Blocked by - 
- [ ] Stylable versioning issue (`^` fetches `2.3.0` and not alpha)
- [ ] Might break wix-ui-intro if [this](https://github.com/wix-private/wix-ui-intro/pull/78) is not merged. 
- [ ] Might break wix-ui-adi if [this](https://github.com/wix-private/wix-ui-adi/pull/321) is not merged.

Better alternative IMO is to set export `StylableDOMUtilCompat` as `StylableDOMUtil` (similar to Unidriver fix) - @barak007 @tomrav - your opinion?

**Comment by NitayRabi** [comment]
Replaced by cleaner and more specific - `https://github.com/wix/wix-ui/pull/1476`
